Dockerfile

```diff
@@ -28,11 +28,12 @@ RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSI
     && tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
     && rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
 
-COPY package.json yarn.lock /usr/src/app/
+COPY . /usr/src/app
```

Comment:
the goal here is to perform yarn install first, without depending on changing to the code to allow better layer caching : hence it is required to copy only package.json and yarn.lock

To deal with the build, you should thus do it in 2 steps here:

```
COPY package.json yarn.lock /usr/src/app/
RUN yarn install ...
COPY . /usr/src/app
RUN yarn build
```